src/main/java/org/folio/circulation/rules/CirculationRulePolicyIdEntity.java

```diff
@@ -0,0 +1,30 @@
+package org.folio.circulation.rules;
+
+import java.util.List;
+
+public class CirculationRulePolicyIdEntity {
+
+  private final String policyId;
+  private final List<String> ruleConditions;
+
+  public CirculationRulePolicyIdEntity(String policyId, List<String> ruleConditions) {
+    this.policyId = policyId;
+    this.ruleConditions = ruleConditions;
+  }
+
+  public String getPolicyId() {
+    return policyId;
+  }
+
+  public List<String> getRuleConditions() {
```

Comment:
Is there a known set of these, the item limit changes seem to rely on 3 specific kinds of conditions?

Reply:
yes, it relies on 3 specific kinds of conditions

Reply:
I think it could be easier to understand this behaviour if these three conditions were explicitly modelled, as named properties, rather than a list of strings (isn’t each condition a pair of values?)

Reply:
Sorry,  don't quite understand what you mean. I only can suggest Enum list instead of String, if I understood you correctly, something like that: 
 ITEM_TYPE("ItemType"),
 LOAN_TYPE("LoanType"),
 PATRON_GROUP("PatronGroup")

Could you please provide an example the better option? Thanks

Reply:
Am I correct in understanding that a condition is both the record type and the value matched for a particular rule?

Reply:
Let me provide example to be more clear:
![image](https://user-images.githubusercontent.com/53909129/71976920-3489a680-3220-11ea-9ca5-2a05dd4ce25d.png)
For the rule that I marked, the conditional List<String> will be the next: "ItemType", "PatronGroup"

Reply:
Hmm, ok, I had understood that the condition would be a pair of the record type and the value. I thought that the value from the matching conditions were being used for filtering the items.

What I'm trying to understand is why use an array of magic strings over a set of booleans or named properties.

If we only want to know about the presence then it could be a boolean per condition (or if we wanted it to be more extensible, an object with a `isPresent` property). If we wanted to know the value, then it could be a property .e.g. getLoanTypeValue in a conditions object, that would be null if not present.

The internal design can be easily evolved, I'm more concerned by the external API design, where I think we a more specific design.

Does that make sense?

Reply:
I've created AppliedRuleConditionsEntity and updated logic. Could you please have a look?